### PR TITLE
Change path to serial hack

### PR DESCRIPTION
**Pull request by wojciechowskidawid**

### CODE REVIEW

File: .github/workflows/generate_ipynb_files.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ jobs:
       GH_SERVICE_ACCOUNT_NAME: "tflite-bot"
       GH_SERVICE_ACCOUNT_EMAIL: "tflite-bot@antmicro.com"
       GH_SERVICE_ACCOUNT_TOKEN: ${{ secrets.GH_SERVICE_ACCOUNT_TOKEN }}
+      CLONE_REPO_SED_CMD: $(sed -i '1s/^/!git clone https:\/\/github.com\/antmicro\/tensorflow-arduino-examples.git .\/ \n/' build.sh)
```
Comment (PiotrZierhoffer) [inline .github/workflows/generate_ipynb_files.yml:17]: can we have something like `echo -e "git clone .... \n$(cat build.sh)" > build.sh` instead? It's so much simpler

Comment (PiotrZierhoffer) [reply .github/workflows/generate_ipynb_files.yml:17]: and maybe add `sed -i 's/^/!/' build.sh` to the same variable. Also rename to `PREPARE_BUILD_SH`

Comment (PiotrZierhoffer) [reply .github/workflows/generate_ipynb_files.yml:17]: will the dot work? Is it a guaranteed empty directory?

File: examples/person-detection/build.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@ curl -fsSL https://raw.githubusercontent.com/arduino/arduino-cli/master/install.
 bin/arduino-cli core install arduino:mbed
 bin/arduino-cli lib install JPEGDecoder@1.8.0
 git clone --quiet https://github.com/ArduCAM/Arduino.git && cp -r Arduino/ArduCAM $HOME/Arduino/libraries
-git clone --quiet https://github.com/antmicro/tensorflow-arduino-examples.git && cp -r tensorflow-arduino-examples/tensorflow $HOME/Arduino/libraries
+cp -r tensorflow $HOME/Arduino/libraries
```
Comment (PiotrZierhoffer) [inline examples/person-detection/build.sh:5]: mkdir

File: examples/magic-wand/build.sh [hunk 3]
```diff
@@ -1,6 +1,6 @@
 curl -fsSL https://raw.githubusercontent.com/arduino/arduino-cli/master/install.sh | sh
 bin/arduino-cli core install arduino:mbed
 bin/arduino-cli lib install Arduino_LSM9DS1@1.1.0
-git clone --quiet https://github.com/antmicro/tensorflow-arduino-examples.git && cp -r tensorflow-arduino-examples/tensorflow $HOME/Arduino/libraries
+cp -r tensorflow $HOME/Arduino/libraries
```
Comment (PiotrZierhoffer) [inline examples/magic-wand/build.sh:4]: mkdir

Comment (PiotrZierhoffer) [reply examples/magic-wand/build.sh:4]: let's have it for consistency

**Review by PiotrZierhoffer — CHANGES_REQUESTED** [review]

File: .github/workflows/generate_ipynb_files.yml [hunk 4]
```diff
@@ -13,6 +14,9 @@ jobs:
       GH_SERVICE_ACCOUNT_NAME: "tflite-bot"
       GH_SERVICE_ACCOUNT_EMAIL: "tflite-bot@antmicro.com"
       GH_SERVICE_ACCOUNT_TOKEN: ${{ secrets.GH_SERVICE_ACCOUNT_TOKEN }}
+      PREPARE_BUILD_SH: |
+        echo -e "git clone https://github.com/antmicro/tensorflow-arduino-examples.git ./ \n" > build.sh
```
Comment (PiotrZierhoffer) [inline .github/workflows/generate_ipynb_files.yml:18]: You're actually overwriting build.sh here.

File: .github/workflows/generate_ipynb_files.yml [hunk 5]
```diff
@@ -5,6 +5,7 @@ on:
     paths:
     - 'examples/*/*.py'
     - 'examples/*/build.sh'
```
Comment (PiotrZierhoffer) [inline .github/workflows/generate_ipynb_files.yml:7]: evidently this requires something like `- '.github/**' as well. The other yml as well

File: .github/workflows/generate_ipynb_files.yml (hunk 4 above)
Comment (PiotrZierhoffer) [reply .github/workflows/generate_ipynb_files.yml:18]: `./` -> `.`

But I DON'T think it works. You can't just clone to `.` if it's a non-empty directory.

**Review by PiotrZierhoffer — CHANGES_REQUESTED** [review]

File: .github/workflows/generate_ipynb_files.yml (hunk 4 above)
Comment (wojciechowskidawid) [reply .github/workflows/generate_ipynb_files.yml]: right, changed default value of TENSORFLOW_PATH in build.sh, so we can clone this repo to tensorflow-arduino-examples

**Pull request merged**